### PR TITLE
Mr 20 covariate value input

### PR DESCRIPTION
**Pull request by JanionNevill**
Added inputs to control covariate value in analysis.

### CODE REVIEW

File: R/covariate_analysis_panel.R [hunk 1]
```diff
@@ -20,14 +17,15 @@ covariate_analysis_panel_ui <- function(id) {
...
+            choices = c("Binary", "Continuous"),
+            width = "120pt"
           )
         ),
         style = "display: inline-block;"
       ),
       div(
         conditionalPanel(
-          condition = "output.inferred_type == 'continuous'",
+          condition = "output.inferred_type == 'Continuous'",
```
Comment (nabury) [inline R/covariate_analysis_panel.R:28]: I'm confused about the logic behind this message "If your data is binary, the only allowed values are 0, 1, and NA"

**Review by nabury — CHANGES_REQUESTED** [review]
Could this all have some comments please? It's taking me a long time to work out what is actually happening in each section.

File: R/covariate_analysis_panel.R (hunk 1 above)
Comment (JanionNevill) [reply R/covariate_analysis_panel.R:28]: I've changed the wording to hopefully make it clearer. Happy to workshop it if you think that it still needs it.

Comment (nabury) [reply R/covariate_analysis_panel.R]: > I've changed the wording to hopefully make it clearer. Happy to workshop it if you think that it still needs it.

That's much clearer

**Review by nabury — APPROVED** [review]
Comments make it much easier to follow the logic of the code. All seems to work correctly and tests passed. Changed a minor typo.

**Pull request merged**